web-docs/docs/02-CHANGELOG.mdx

```diff
@@ -5,6 +5,11 @@ sidebar_label: Change Log
 toc_max_heading_level: 2
 ---
 
+## Version 1.0.4
+### Bug Fixes
+- Update Extension library to v1.16.2
+- Fixes issue where local timezone offset is only calculated at startup
```

Comment:
Not the end of the world, but `Update Extension library to v1.16.2` on its own isn't really a bug fix. 

I think that could be a sub-bullet underneath the 'Fixes issue ...' line (or the other way around)

Reply:
+1 on the sub bullet

Reply:
Changed to sub bullet